tests/test_core.py

```diff
@@ -838,8 +838,11 @@ def test_functions():
     imageio.mimsave(fname5, [im[:, :, 0] for im in ims])
     imageio.mimsave(fname5, ims)
     assert os.path.isfile(fname5)
-    
-    # Test volread()
+
+    if not sys.platform.startswith('linux'):
+        return
```

Comment:
Why is this? Does this not work on osx? If it does not, maybe `if not sys.platform.startswith('darwin'):` because we do test on windows.